src/Components/Users/UserAdd.tsx

```diff
@@ -597,14 +594,18 @@ export const UserAdd = (props: UserProps) => {
     };
   };
 
+  if (isLoading) {
+    return <Loading />;
+  }
+
   return (
     <Page
       title={headerText}
       options={
         <Link
-          href="https://school.coronasafe.network/targets/12953"
+          href="https://school.coronasafe.network/targets/12953 "
           className="inline-block rounded border border-gray-600 bg-gray-50 px-4 py-2 text-gray-600 transition hover:bg-gray-100"
-          target="_blank"
+          target="_blank "
```

Comment:
Why was this changed?
```suggestion
          target="_blank"
```